changes/1173.bugfix.md

```diff
@@ -0,0 +1 @@
+Use right roles in `objects.inv` file, in documentation, so `intersphinx` can normally access them.
```

Comment:
No need for a changelog fragment btw :)

Reply:
Deleted in dab0dd69aa46559865302a67adf0e0b279a18445.